### PR TITLE
Updated the “test_calcpmvpdd.R” and tested

### PR DESCRIPTION
**Pull request by number9527-12**
### screenshot
image

### CODE REVIEW

**Comment by number9527-12** [comment]
### The screenshot for test_calaSET.R
image

**Comment by number9527-12** [comment]
### screenshot for test_calcePMV.R
image

**Comment by number9527-12** [comment]
### screenshot test_calcUtci
image

**Comment by number9527-12** [comment]
### screenshot for calcATHBstrandard
image

File: tests/testthat.R [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# This file is part of the standard setup for testthat.
```
Comment (TwinGan) [inline tests/testthat.R]: don't delete this file

File: tests/testthat/test_calUtci.R [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      vel = vel,
+      rh = rh
+    )
+    
+    expected_utci <- outputs[[1]]
+
+    result <- as.numeric(result)
+    expected_utci <- as.numeric(expected_utci)
+
+    print(paste("Test case", i, "- Result:", result, "Expected:", expected_utci))
```
Comment (TwinGan) [inline tests/testthat/test_calUtci.R]: do not need to keep the print

File: tests/testthat/test_calcSET.R [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  tolerance_set <- tolerance$set
+  total_cases <- nrow(data_list)
+
+  for (i in seq_len(total_cases)) {
+    inputs <- as.list(data_list$inputs[i, ])
+    outputs <- as.list(data_list$outputs[i, ])
+
+    units <- ifelse(is.null(inputs$units) || is.na(inputs$units), "SI", inputs$units)
+
+    if (units == "IP") {
```
Comment (TwinGan) [inline tests/testthat/test_calcSET.R:20]: maybe we don't neet to run units = IP test cases

**Review by TwinGan — APPROVED** [review]
pass

**Review by TwinGan — APPROVED** [review]
pass

**Pull request merged**